src/v1/interfaces/IQueryManager.sol

```diff
@@ -47,6 +47,23 @@ interface IQueryManager {
         uint256 endBlock
     ) external payable returns (uint256);
 
+    /// @notice Submits a new request to the registry.
```

Comment:
Can you write down the difference with the previous request method is that this one is specifically made for tabular queries that return a list of rows. For aggregated queries (AVG COUNT etc) prefer to use `request`.

It's also a debate we should have as to how we should shape the API down the line because extending that forever will be hard (for GROUP BY etc) but hopefully we can keep most of the same API...